graphql/client.go

```diff
@@ -75,65 +74,64 @@ type Doer interface {
 	Do(*http.Request) (*http.Response, error)
 }
 
-type payload struct {
+// Payload contains all the values required to build queries executed by
+// the graphql.Client.
+type Payload struct {
```

Comment:
Now that this is public, we probably ought to name it `Request`?  Since that's what it is.  Not sure why I originally called in `Payload`.

Reply:
That sounds reasonable. Changed it.